web-client/public/js/constants.js

```diff
@@ -1,2 +1,23 @@
 export const GREY_EDGES_DASHED_MENU = "#grey-edges-dashed-menu";
 export const GREY_EDGES_DASHED_SIDEBAR = "#dashedGrayLineButton";
+
+// Demo Stuff
+const UNWEIGHTED_DEMO_ID   = "#unweighted";
+const UNWEIGHTED_DEMO_PATH = "demo/unweighted";
+const UNWEIGHTED_DEMO_NAME = "Demo #1: Unweighted GRN (21 genes, 50 edges)";
+const WEIGHTED_DEMO_ID     = "#weighted";
+const WEIGHTED_DEMO_PATH   = "demo/weighted";
+const WEIGHTED_DEMO_NAME   = "Demo #2: Weighted GRN (21 genes, 50 edges, Dahlquist Lab unpublished data)";
+const SCHADE_INPUT_ID      = "#schadeInput";
+const SCHADE_INPUT_PATH    = "demo/schadeInput";
+const SCHADE_INPUT_NAME    = "Demo #3: Unweighted GRN (21 genes, 31 edges)";
+const SCHADE_OUTPUT_ID     = "#schadeOutput";
+const SCHADE_OUTPUT_PATH   = "demo/schadeOutput";
+const SCHADE_OUTPUT_NAME   = "Demo #4: Weighted GRN (21 genes, 31 edges, Schade et al. 2004 data)";
+
+export const DEMO_INFORMATION = [
+    [ WEIGHTED_DEMO_ID,   WEIGHTED_DEMO_PATH,   WEIGHTED_DEMO_NAME   ],
+    [ UNWEIGHTED_DEMO_ID, UNWEIGHTED_DEMO_PATH, UNWEIGHTED_DEMO_NAME ],
+    [ SCHADE_INPUT_ID,    SCHADE_INPUT_PATH,    SCHADE_INPUT_NAME    ],
+    [ SCHADE_OUTPUT_ID,   SCHADE_OUTPUT_PATH,   SCHADE_OUTPUT_NAME   ]
+];
```

Comment:
Imagine, all of these constants were in _upload.js_ but they served a single purpose. Nice to have them set aside cleanly as  a bunch of constants now. Note also that the only actual export needed was `DEMO_INFORMATION`—the rest are actually "private" constants. So again this helps clean the code up and clarify what is shared vs. not.